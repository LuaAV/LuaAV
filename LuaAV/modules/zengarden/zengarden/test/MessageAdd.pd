#N canvas 761 150 450 300 10;
#X obj 21 19 loadbang;
#X msg 140 71 13;
#X obj 21 174 print;
#X msg 111 71 x;
#X msg 80 71 3;
#X msg 51 71 4;
#X msg 21 71 5 7;
#X msg 170 71 x;
#X msg 200 71 5;
#X obj 21 40 t b b b b b b b b b;
#X msg 233 71 2 3 4;
#X msg 280 71 5;
#X obj 21 153 + 5;
#X connect 0 0 9 0;
#X connect 1 0 12 0;
#X connect 3 0 12 0;
#X connect 4 0 12 1;
#X connect 5 0 12 0;
#X connect 6 0 12 0;
#X connect 7 0 12 1;
#X connect 8 0 12 0;
#X connect 9 0 6 0;
#X connect 9 1 5 0;
#X connect 9 2 4 0;
#X connect 9 3 3 0;
#X connect 9 4 1 0;
#X connect 9 5 7 0;
#X connect 9 6 8 0;
#X connect 9 7 10 0;
#X connect 9 8 11 0;
#X connect 10 0 12 0;
#X connect 11 0 12 0;
#X connect 12 0 2 0;
