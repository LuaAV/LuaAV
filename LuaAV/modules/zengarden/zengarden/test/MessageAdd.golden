[@ 0.000ms] print: 10
[@ 0.000ms] print: 5
[@ 0.000ms] print: 8
[@ 0.000ms] print: 16
[@ 0.000ms] print: 7
[@ 0.000ms] print: 12
